### DISCUSSION TITLE
Problème d'affichage après MAJ 2.02

### DISCUSSION THREAD

**Discussion by RVDROU** [robot-soccer-kit/robot-soccer-kit#98] [Q&A] [answered]
Après la mise à jour du game controller pour la version 2.0.2 , l'affichage du signal vidéo du GC s'est arrêté avec un comportement particulier. Le calibrage du terrain reste opérationnel (message vert à droite de la vidéo) et se perd lorsque l'on modifie le paramétrage vidéo, ce qui semble logique. PAr contre j'ai juste une image fixe du terrain qui s'affiche, aucun robot n'est détecté alors que la balle est bien repérée (ses coordonnées sont présentes) et la vitesse d'acquisition vidéo tombe à 5 fps. J'ai bien sûr vérifié le câblage de la caméra et redémarré l'ordi au cas où.
Un problème similaire a été signalé par Emmanuel sur un message l'année dernière. Etant moi aussi sur Linux Mint j'ai refait une installation propre en Linux Ubuntu Mate 22.04 en suivant pas à pas la démarche du github mais le problème persiste.
Avez vous une solution à me proposer, je suis à court d'idée.
Merci d'avance.

Hervé Drougard - Périgueux -

**Comment by Gregwar** [comment, marked as answer]
Je viens de faire quelques tests et correctifs, pouvez-vous essayer la version `2.0.3` ?

Si le problème persiste:
- Pouvez vous supprimer `config.json` et relancer
- Y'a t-il un message d'erreur qui apparaît dans le terminal ?
- Pouvez vous fournir la sortie de la commande `pip freeze` ?
- Est-ce que le même problème apparaît sur `1.1.1` par exemple ?

**Reply by RVDROU** [reply]
Merci pour cette réponse rapide. Je n'ai pas le serveur sous la main durant le WE, je ferai les tests dès mardi matin. Pour répondre à votre dernière question, le serveur en 1.1.1 fonctionnait parfaitement sur la machine sous Mint. Le problème a apparu après la mise à jour de la V2.02 et a persisté malgré une réinstallation complète du serveur pour une version Ubuntu Mate 22.04.

**Comment by RVDROU** [comment]
Le problème venait visiblement du disque dur de la machine ou de la mise à jour du game controler. Après réinstallation complète la vidéo fonctionne correctement maintenant.
Merci pour ces conseils.
